src/transport/raw/MessageHeader.h

```diff
@@ -42,38 +42,49 @@
 
 namespace chip {
 
+static constexpr size_t kMaxTagLen = 16;
+
 namespace detail {
 // Figure out the max size of a packet we can allocate, including all headers.
-static constexpr size_t kMaxIPPacketSizeBytes       = 1280;
+static constexpr size_t kMaxUdpIPPacketSizeBytes       = 1280;
+static constexpr size_t kMaxTcpIPPacketSizeBytes       = 12800;
```

Comment:
This is not big enough for pictures

Reply:
Correct, we haven't really decided how big it should get for pictures/snapshots
if you do have a suggestion we can start by that number and later we can update it